bridge/opentracing/wrapper.go

```diff
@@ -24,21 +25,62 @@ import (
 // WrapperTracerProvider is an OpenTelemetry TracerProvider that wraps an
 // OpenTracing Tracer.
 type WrapperTracerProvider struct {
-	wTracer *WrapperTracer
+	getWrappedTracer func(name string, opts ...trace.TracerOption) *WrapperTracer
 }
 
 var _ trace.TracerProvider = (*WrapperTracerProvider)(nil)
 
-// Tracer returns the WrapperTracer associated with the WrapperTracerProvider.
-func (p *WrapperTracerProvider) Tracer(_ string, _ ...trace.TracerOption) trace.Tracer {
-	return p.wTracer
+// Tracer creates a WrapperTracer associated with the WrapperTracerProvider.
+func (p *WrapperTracerProvider) Tracer(name string, opts ...trace.TracerOption) trace.Tracer {
+	return p.getWrappedTracer(name, opts...)
 }
 
+// Deprecated: Use NewTracerProvider(...) instead.
+//
```

Comment:
This needs to be at the bottom of the documentation block: https://rakyll.org/deprecated/

Reply:
Fixed!